src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -650,6 +650,10 @@ $facility-locator-color-gray: #ccc;
   text-decoration: none;
 }
 
+.facility-phone-link {
```

Comment:
Can we use values from https://design.va.gov/utilities/margins#margin-left instead?